lib/exandra.ex

```diff
@@ -63,6 +63,18 @@ defmodule Exandra do
       field :home_phone, Exandra.UDT, type: :phone_number
       field :office_phone, Exandra.UDT, type: :phone_number
 
+  NOTE: There is no validation with Exandra.UDT and the keys _must_ be strings.
```

Comment:
My favorite thing about Earth is now ExDoc annotation blocks.

```suggestion
  > #### String Keys {: .warning}
  >
  > There is no validation with `Exandra.UDT` and the keys _must_ be strings.
```